specification/vmwarecloudsimple/resource-manager/readme.go.md

```diff
@@ -0,0 +1,19 @@
+## Go
```

Comment:
File is incorrect, please see https://github.com/Azure/azure-rest-api-specs/blob/master/specification/batch/resource-manager/readme.go.md for a template.